src/collectors/kafkastat/test/testkafka.py

```diff
@@ -108,10 +108,10 @@ def test_query_mbean(self, get_mock):
         get_mock.return_value = self._get_xml_fixture('mbean.xml')
 
         expected_metrics = {
-            'kafka.logs.mytopic-1.CurrentOffset': long('213500615'),
-            'kafka.logs.mytopic-1.NumAppendedMessages': long('224634137'),
-            'kafka.logs.mytopic-1.NumberOfSegments': int('94'),
-            'kafka.logs.mytopic-1.Size': long('50143615339'),
+            'kafka.kafka.logs.mytopic-1.CurrentOffset': long('213500615'),
```

Comment:
Why add `kafka` again in the block of metrics? (I mean on the collector side since i realize this is the test file)